crates/re_query/src/dataframe_util.rs

```diff
@@ -127,6 +128,57 @@ where
     Ok(DataFrame::new(vec![series0, series1, series2])?)
 }
 
+pub fn df_builder1_arch<
+    'a,
+    C0: re_types::Component + Clone + Into<::std::borrow::Cow<'a, C0>> + 'a,
+>(
+    c0: &[Option<C0>],
+) -> crate::Result<DataFrame> {
+    let array0 = C0::try_to_arrow_opt(c0.to_owned(), None)?;
```

Comment:
Hmm you shouldn't be forced to go for an owned copy here, that's a failure on the Component trait's part; I'll have a look at some point.